### PR TITLE
Fix uxp symbol

### PR DESCRIPTION
**Pull request by sjillen**
This will NOT change the URL to `DAO/UXP` when we navigate through the app. 
Governance-ui takes the DAO Symbol as community mint, meaning it should be UXP and not UXD (which makes sense).
If we want to keep UXD in the URL but have UXP display the comments for the vote weight, it's doable but will break when we try to fork back. -> found a way : use nextjs redirect

### CODE REVIEW

**Review by acamill — APPROVED** [review]
Ok thanks! this mean it will break the references to it in the tweets and all? Maybe wait for this vote to be over before merging

**Comment by sjillen** [comment]
yes, it will break the link of the tweet. Will merge after vote is done
**edit**: it won't break anymore

**Review by 0xCryptoSheik — COMMENTED** [review]
LGTM thnx

**Pull request merged**
